client/my-sites/plans/jetpack-plans/product-grid/utils.ts

```diff
@@ -92,7 +93,11 @@ export const getProductsToDisplay = ( {
 		// Remove add-on products
 		.filter( removeAddons )
 		// Remove products that don't match the selected duration
-		.filter( ( product ): product is SelectorProduct => product?.term === duration )
+		// However, allow products that ONLY have a monthly term to come through in either view
+		.filter(
```

Comment:
Do you think this could be slightly improved by having a constant named something like `JETPACK_SINGLE_TERM_PRODUCTS` or `JETPACK_SINGLE_TERM_OPTION_PRODUCTS` so we can allow both products with only a monthly term and products with only a yearly term through? 

This could be seen as YAGNI, but I think since we are generalizing it to all "Monthly only" products here even though we only have one, it could be good to generalize it even further since the only effort involved is renaming the variable for now, or perhaps creating an empty `JETPACK_YEARLY_ONLY_PRODUCTS` array and combining it with the `JETPACK_MONTHLY_ONLY_PRODUCTS` to use here